insights/specs/__init__.py

```diff
@@ -114,7 +114,7 @@ class Specs(SpecSet):
     dmsetup_info = RegistryPoint()
     dnf_modules = RegistryPoint()
     dnf_module_list = RegistryPoint()
-    dnf_module_info = RegistryPoint(multi_output=True)
+    dnf_module_info = RegistryPoint()
```

Comment:
@csams - Please have a look to this one.

I'm sure it is **NG** to remove the `multi_output=True` from a [`foreach_execute` spec](https://github.com/RedHatInsights/insights-core/pull/2110/commits/ee8ac0049746d480c7fdde36ddf11f6c7cd6b47c#diff-40c874757ab5b5f019039c435a8334daR231) as below:

```python
dnf_module_info = foreach_execute(dnf_module_names, "/usr/bin/dnf -C --noplugins module info %s", deps=[IsRhel8])
```
The spec `dnf_module_info` depends on another datasource `dnf_module_names`.

According to the above [suggestion](https://github.com/RedHatInsights/insights-core/pull/2110#discussion_r307358091) from @Rezney, in order to get a better execution efficiency, I just want the provider datasource `dnf_module_names` to return a single string which contains all the module names, instead of a list of separated module names. However, the `foreach_execute` will lead the parser `DnfModuleInfo` being wrapped into a list, which brings a bit complicated when using the parser. And we have to write a combiner to make its usage simple.

Can we do the following enhancement to the `simple_command` to let it be able to receive a string (but not a list of strings) generated by another datasource? and then the parser result wouldn't be wrapped anymore.

```python
@datasource(DnfModuleList)
def dnf_module_names(broker):
    dml = broker[DnfModuleList]
    if dml:
        return (' ').join([m for m in dml])
    raise SkipComponent()

dnf_module_info = simple_command("/usr/bin/dnf -C --noplugins module info %s", dnf_module_names, deps=[IsRhel8])
```
I have implemented a version of code for this idea, please have a look as well.
- https://github.com/RedHatInsights/insights-core/pull/2112